app/src/main/res/layout/activity_main.xml

```diff
@@ -21,12 +22,14 @@
     <fragment
         android:id="@+id/nav_host_fragment"
         android:name="androidx.navigation.fragment.NavHostFragment"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_width="406dp"

```

Comment:
remove these fixed values

Reply:
Actually, I don't think you can remove this. Change it to 0dp to indicate that the size is being changed somewhere else.

Reply:
Even better use "match_parent" again

Reply:
Using match_parent might be an issue when we retrieve the album art. Might stretch the image

Reply:
match_parent works fine for the width. It doesn't work well for the height